TESTING.asciidoc

```diff
@@ -161,6 +161,11 @@ Don't count hypercores for CPU-intense tests and leave some slack
 for JVM-internal threads (like the garbage collector). Make sure there is
 enough RAM to handle child JVMs.
 
+You can set the default number of JVMs used in `~/.gradle/gradle.properties`:
```

Comment:
By default this shouldn't be necessary? This is only for overriding what "auto" configures in randomized testing (usually needed b/c it rounds down, so if you have eg 6 cores like I do on a box, you get 4).